app/views/partners/profiles/step/edit.html.erb

```diff
@@ -0,0 +1,114 @@
+<section class="content-header">
+  <div class="container-fluid">
+    <div class="row mb-2">
+      <div class="col-sm-6">
+        <% content_for :title, "Step Editing - #{current_partner.name}" %>
+        <h1><i class="fa fa-edit"></i>&nbsp;&nbsp;Edit My Organization&nbsp;&nbsp;&nbsp;
+          <%= partner_status_badge(current_partner) %>
+          <small>for <%= current_partner.name %></small>
+        </h1>
+      </div>
+      <div class="col-sm-6">
+        <ol class="breadcrumb float-sm-right">
+          <li class="breadcrumb-item"><a href="<%= partner_user_root_path %>"><i class="fa fa-home fa-lg"></i></a></li>
+          <li class="breadcrumb-item">
+            <a href="<%= edit_partners_profile_path %>"><%= "#{current_partner.name}" %></a></li>
+          <li class="breadcrumb-item"><a href="#">Edit</a></li>
+        </ol>
+      </div>
+    </div>
+  </div><!-- /.container-fluid -->
+</section>
+
+<div class="alert alert-info mx-5 mt-3" role="alert">
+  Instructions: Please fill out the following form sections carefully. Ensure that all required fields are completed.
+</div>
+
+<div data-controller="accordion">
+  <%= simple_form_for current_partner,
+                      data: { controller: "form-input", accordion_target: "form" },
+                      url: partners_profile_path,
+                      html: { multipart: true } do |f| %>
+
+    <%= render 'partners/profiles/step/form_actions', f: f, partner: current_partner %>
+
+    <div class="accordion mx-5 mt-3" id="accordionExample">
+
+      <%# Agency Information %>
+      <div class="accordion-item">
+        <h2 class="accordion-header">
+          <button class="accordion-button <%= section_with_errors?('agency_information', @sections_with_errors) ? '' : 'collapsed' %>" type="button" data-bs-toggle="collapse" data-bs-target="#agency_information" aria-expanded="<%= section_with_errors?('agency_information', @sections_with_errors) %>" aria-controls="agency_information">
+            <div class='d-flex justify-content-between align-items-center w-100'>
+              <div class='fs-4'>
+                <i class="fa fa-edit mr-2"></i>Agency Information
+              </div>
+            </div>
+          </button>
+        </h2>
+        <div id="agency_information" class="accordion-collapse collapse <%= section_with_errors?('agency_information', @sections_with_errors) ? 'show' : '' %>">
+          <div class="accordion-body">
+            <%= render 'partners/profiles/step/agency_information_form', f: f, partner: current_partner, profile: current_partner.profile %>
+          </div>
+        </div>
+      </div>
```

Comment:
Can this be extracted into a partial with relevant locals (name, partial ID etc.)? Partial renders can take blocks, so if you need specific HTML within the big skeleton you can do that if necessary.

Reply:
Extracted to partial.